### PR TITLE
Docs: prepare for release

### PR DESCRIPTION
**Pull request by nguyentoanit**

### CODE REVIEW

**Comment by moltar** [comment]
Task linked: [GHORG-43 Prepare for release](https://app.clickup.com/t/2250003/GHORG-43)

File: package.json [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
     "name": "Roman Filippov",
     "email": "roman@scaleleap.com",
     "url": "https://www.scaleleap.com/"
   },
+  "contributors": [
+    {
+      "name": "Toan Nguyen",
+      "url": "https://github.com/nguyentoanit"
+    }
+  ],
```
Comment (moltar) [inline package.json:15]: What about me? 😢

File: package.json [hunk 2]
```diff
@@ -72,7 +78,13 @@
     "typescript": "4.2.3"
   },
   "keywords": [
-    "template",
+    "amazon mws",
+    "amazon selling partner",
+    "amazon sp api",
+    "nodejs",
+    "selling partner api",
```
Comment (moltar) [inline package.json:85]: duplicate

**Review by moltar — CHANGES_REQUESTED** [review]
Also need a `LICENSE` file, can copy from MWS repo.

**Comment by nguyentoanit** [comment]
> Also need a `LICENSE` file, can copy from MWS repo.

We had a `LICENSE` file already [in our repo](https://github.com/ScaleLeap/selling-partner-api-sdk/blob/master/LICENSE).

**Comment by moltar** [comment]
> > Also need a `LICENSE` file, can copy from MWS repo.
> 
> We had a `LICENSE` file already [in our repo](https://github.com/ScaleLeap/selling-partner-api-sdk/blob/master/LICENSE).

Ah, ok, good 😁

**Review by moltar — APPROVED** [review]

**Pull request merged**